src/spec/orderexport.spec.coffee

```diff
@@ -109,6 +112,7 @@ describe 'OrderExport', ->
 
     @orderExport.runCSVAndStreamToFile(callback)
       .then () =>
+        expect(data).toEqual([ 'id,customerEmail', 'aaa,\nbbb,\nccc,' ])
```

Comment:
I removed new line delimiter from header [here](https://github.com/sphereio/sphere-order-export/pull/55/files#diff-cc1cc3d970aba69d45634a03eb104827R70) and added it [here](https://github.com/sphereio/sphere-order-export/pull/55/files#diff-6f8ff9223bf46905bc083826f4c04c70R115) so the new line is printed out after every chunk (csv header, csv data) which is being printed out.

In this 2 items array the new line would be added after every item.